modules/installation-azure-preparing-diskencryptionsets.adoc

```diff
@@ -0,0 +1,144 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_azure/enabling-disk-encryption-sets-azure.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-disk-encryption-sets"]
+= Preparing an Azure Disk Encryption Set
+The {product-title} installer can use an existing Disk Encryption Set with a user-managed key. To enable this feature, you can create a Disk Encryption Set in Azure and provide the key to the installer.
+
+.Procedure
+
+. Set the following environment variables for the Azure resource group:
++
+[source,terminal]
+----
+$ export RESOURCEGROUP="<resource_group>"
+----
++
+[source,terminal]
+----
+$ export LOCATION="<location>"
+----
++
+where:
+
+`<resource_group>`:: Specifies the name of the Azure resource group where you will create the Disk Encryption Set and encryption key. You should create the Disk Encryption Set in a different resource group than the resource group where you install the cluster to avoid losing access to your keys after the cluster is destroyed.
+`<location>`:: Specifies the Azure location where you will create the resource group.
++
+. Set the following environment variables for the Azure Key Vault and Disk Encryption Set:
++
+[source,terminal]
+----
+$ export KEYVAULT_NAME="<keyvault_name>"
+----
++
+[source,terminal]
+----
+$ export KEYVAULT_KEY_NAME="<keyvault_key_name>"
+----
++
+[source,terminal]
+----
+$ export DISK_ENCRYPTION_SET_NAME="<disk_encryption_set_name>"
+----
++
+. Set the environment variable for the ID of your Azure Service Principal:
++
+[source,terminal]
+----
+$ export CLUSTER_SP_ID="<service_principal_id>"
+----
++
+where:
```

Comment:
Same as above comment. Seeing this example with more steps, I'm leaning towards option 1: 
```
[source,terminal]
----
$ export KEYVAULT_NAME... \ <1>
	KEYVAULT_KEY... \ <2>
	DISK_ENCRYPTION... \ <3>
----
+
<1>
<2>
<3>
```